src/components/Navbar.js

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import "./Navbar.css";
+import { Menu } from "antd";
+import "./Navbar.css";
+import {
+  PhoneOutlined,
+  StarOutlined,
+  HomeOutlined,
+  MailOutlined,
+  ShopOutlined,
+  ReadOutlined,
+  AppstoreOutlined,
+  ShoppingOutlined
+} from "@ant-design/icons";
+
+const { SubMenu } = Menu;
+
+class Navbar extends React.Component {
+  state = {
+    current: 'mail',
+  };
+  
+  handleClick = (e) => {
+    console.log("click ", e);
+    this.setState({ current: e.key });
+  };
+
+  render() {
+    const { current } = this.state;
+    return (
+      <div className="navbar">
+        <nav className="navbar-top">
+          <div>
+            <PhoneOutlined /> +880 1234 123456
+          </div>
+          <div style={{fontSize: "2rem"}} className="site-name">PLUTO</div>
+          <div>Log in&nbsp;&nbsp;|&nbsp;&nbsp;<ShoppingOutlined /></div>
+        </nav>
+        <hr className="nav-hr" />
+        <Menu
+          onClick={this.handleClick}
+          mode="horizontal"
+          className="navbar-bottom"
+        >
+          <Menu.Item key="Home" icon={<HomeOutlined />}>
+            Home
+          </Menu.Item>
+
+          <SubMenu key="Shop" icon={<ShopOutlined />} title="Shop">
```

Comment:
Can you render the sub-menus using a list? So we can render it dynamically later on?

Reply:
In ant design, the sub-menus are like the customized version of `<li>`. So, yes, we can use `<li>` to render sub-menus. But then those `<li>` items must be styled with `css`.
This is what happens when I use `<li>`to render sub-menus without using any `css`.

![2021-01-20-10-16-13](https://user-images.githubusercontent.com/32035481/105126317-a7eb1700-5b08-11eb-9c53-1aae58e3ba18.png)
![Screenshot (83)](https://user-images.githubusercontent.com/32035481/105126453-f6001a80-5b08-11eb-9b3b-8eea1f254c12.png)

As for dynamic rendering, I think sub-menus can be dynamically rendered.

Reply:
Can't you just loop over `Menu.Item` ? Why you need to use raw `li`?

Reply:
I am not understanding clearly why do we need to loop over `Menu.item` to render those dynamically. But as for dynamically rendering different navbar based whether the user is logged in or not, I can do that.